packages/playground/src/components/main_footer.vue

```diff
@@ -36,20 +36,23 @@
           <div>
             <a
               color="error"
-              href="https://manual.grid.tf/threefold_token/buy_sell_tft/buy_sell_tft.html"
+              href="https://www.manual.grid.tf/documentation/threefold_token/buy_sell_tft/buy_sell_tft.html"
               class="app-link text-subtitle-2"
               target="_blank"
               >Buy the Token</a
             >
           </div>
           <div>
-            <a href="https://manual.grid.tf/farmers/farmers.html" class="app-link text-subtitle-2" target="_blank"
+            <a
+              href="https://www.manual.grid.tf/documentation/farmers/farmers.html"
+              class="app-link text-subtitle-2"
+              target="_blank"
               >Start Farming</a
             >
           </div>
           <div>
             <a
-              href="https://library.threefold.me/info/threefold#/threefold__governance"
+              href="https://www.manual.grid.tf/knowledge_base/about/threefold_history.html"
```

Comment:
What does Threefold history have to do with Governance?

Reply:
@Mik-TF Please explain as this change was made by you in the referenced PR.

Reply:
We didn't have exactly the Governance page on manual so I took one page that fit most of the content. But I can just make a PR on info_grid and add the library's governance page, then we can link the proper governance page, but from the manual instead of library. Will work on this today to send quickly.

Reply:
New link: https://www.manual.grid.tf/knowledge_base/about/governance.html

Reply:
The PR has passed. It's now on manual.grid.tf: https://manual.grid.tf/knowledge_base/about/governance.html

Reply:
updated @zaelgohary, thanks @Mik-TF